README.md

```diff
@@ -52,32 +52,32 @@ end
 ```
 
 ### Configure it
-Set these mandatory options in the rails_helper:
+Set these optiional options in the rails_helper:
 
 | Option | Value | Description |
 | -- | -- | -- |
-| body_file_path | Pathname instance or fullpath string | Json file that will include openapi version, the basic information about the api, api title and api version. |
-| desc_folder_path | Pathname instance or fullpath string | Folder with markdown descriptions. |
+| descriptions_location | Pathname instance or fullpath string | Folder with markdown descriptions. |
```

Comment:
We need to set this only when we have some descriptions? If so, we should make that clear in the documentation.
Or is there a default location?